packages/ssx-sdk/src/modules/Storage.ts

```diff
@@ -1,11 +1,29 @@
+import {
+  OrbitConnection,
+  activateSession,
+  hostOrbit,
+  wasmPromise,
+  Request,
+} from 'kepler-sdk';
+import { initialized } from '@spruceid/ssx-sdk-wasm';
+
 import { openDB, IDBPDatabase } from 'idb';
 import { IEncryption } from './Encryption';
-import { IUserAuthorization } from './UserAuthorization';
+import {
+  IUserAuthorization,
+  UserAuthorizationConnected,
+  SiweMessage,
+} from '../';
+import {
+  ConfigOverrides,
+  SSXClientSession,
+  SSXExtension,
+} from '@spruceid/ssx-core/client';
 
 /**
  * Represents a storage interface that defines basic storage operations.
  */
-interface IStorage {
+interface IStorage extends SSXExtension {
```

Comment:
Maybe `KVStorage`?

Reply:
Open to this change. What do others think of changing from Storage to KVStorage for all instances of Storage?